.github/workflows/stale-management.yml

```
@@ -0,0 +1,358 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Archive old feature branches
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { owner, repo } = context.repo;
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)